src/services/PurchaseOrderService.ts

```diff
@@ -9,14 +9,15 @@ const fetchPurchaseOrders = async (payload: any): Promise <any>  => {
   });
 }
 
-const fetchPODetails = async (query: any): Promise <any> => {
+const fetchPODetails = async (payload: any): Promise <any> => {
```

Comment:
I think this is an unwanted change, we can again make it query, WDYT?

Reply:
As I think that earlier when you have written down code, you may have thought that data which we will be sending will come as query, but now I have implemented it as payload so that I think we should go with payload, not query.